src/xrpld/app/tx/detail/VaultClawback.cpp

```diff
@@ -286,14 +287,27 @@ VaultClawback::doApply()
     // Keep MPToken if holder is the vault owner.
     if (holder != vault->at(sfOwner))
     {
-        if (auto const ter =
-                removeEmptyHolding(view(), holder, sharesDestroyed.asset(), j_);
-            isTesSuccess(ter))
+        auto const ter =
+            removeEmptyHolding(view(), holder, sharesDestroyed.asset(), j_);
+        if (isTesSuccess(ter))
```

Comment:
Even with the `else if`, `ter` is only used in the context of the `if`, so you can put it back.
```suggestion
        if (auto const ter =
                removeEmptyHolding(view(), holder, sharesDestroyed.asset(), j_);
            isTesSuccess(ter))
```

Reply:
thanks, implemented in https://github.com/XRPLF/rippled/pull/5652/commits/6893015759ad8b7041611551c453b6efd9ac9051